run/sigterm-handler/Dockerfile

```diff
@@ -0,0 +1,47 @@
+# Copyright 2021 Google LLC
```

Comment:
I think it's worth considering omitting Dockerfiles now that the Buildpacks work correctly for these kinds of apps. (No need to address here)

Reply:
I agree. We are starting to think about changes across all samples.